Wyam/theme/_Index.cshtml

```diff
@@ -1,3 +1,5 @@
-@using Endjin.AzureWeekly.WyamModules;
-
-@Html.Raw(Model.Get<Newsletter>("NewsletterContent").IssueNumber + ": " + Model.Get("NewsletterTo") + " - " + Model.Get("NewsletterFrom"))
\ No newline at end of file
+@using Endjin.AzureWeekly.WyamModules;
+
+@Html.Raw(Model.Get<Newsletter>("NewsletterContent").IssueNumber + ": " + Model.Get("NewsletterTo") + " - " + Model.Get("NewsletterFrom"))
+
+@RenderBody()
```

Comment:
Razor either needs a `RenderBody()` or `IgnoreBody()` call - the later would have worked here just fine since the document content is empty, but I usually prefer `RenderBody()` in case I add document content later and want to render it out